sql/moz-fx-data-shared-prod/telemetry/unified_metrics/view.sql

```diff
@@ -8,3 +8,4 @@ FROM
   `moz-fx-data-shared-prod.telemetry_derived.unified_metrics_v1`
 WHERE
   normalized_app_name != 'Focus Android Glean'
```

Comment:
Maybe this is a bit easier to follow?

```suggestion
  normalized_app_name NOT IN ('Focus Android Glean', 'Focus Android Glean BrowserStack')
```

Reply:
I actually separated to make it clear.
Also I tend to avoid the 'NOT IN' clause because it has a slightly more expensive query plan.